plenum/test/monitoring/test_revival_spike_resistant_ema_throughput_measurement.py

```diff
@@ -137,9 +164,11 @@ def test_rsr_ema_tm_after_start_switches_to_revival_on_not_empty_window(tm_after
 
     # [60, 75)
     throughput = tm.get_throughput(62)
+    tm.add_request(62)
```

Comment:
The last two lines should be swapped because the test should verify `get_throughput` return value in REVIVAL, not FADED state.

Reply:
Done.